Makefile

```diff
@@ -8,7 +8,7 @@ integration:
 	mvn -Dskip.surefire.tests -DskipITs=false failsafe:integration-test@failsafe-execution
 
 validate:
-	mvn javadoc:javadoc@javadoc-execution
+	#mvn javadoc:javadoc@javadoc-execution
```

Comment:
this is something I should tack onto the kotlin PR, but unfortunately, the mvn javadoc plugin doesn't play well with kotlin. This article (https://kotlinlang.org/docs/reference/using-maven.html#generating-documentation) suggests using dokka? instead of the javadoc plugin with maven, haven't tried that out yet.

Reply:
could the javadoc plugin run directory-based (or some other way to sort out .kt vs .java—it seems odd that the plug-in would even bother with .kt files)? (some option like `<dependencySourceExcludes>` (though maybe not that one specifically)?)

Reply:
unfortunately, its not failing when trying to process the Kotlin files, it fails when a Java class imports a Kotlin class, and fails with a package not found: 
```
[ERROR] Exit code: 1 - /Users/sbabu/Workspaces/mt-dynamo/src/main/java/com/salesforce/dynamodbv2/mt/mappers/MtAmazonDynamoDb.java:8: error: package com.salesforce.dynamodbv2.mt.sample does not exist
[ERROR] import com.salesforce.dynamodbv2.mt.sample.SampleDataClass;
[ERROR]                                           ^
[ERROR] /Users/sbabu/Workspaces/mt-dynamo/src/main/java/com/salesforce/dynamodbv2/mt/mappers/MtAmazonDynamoDb.java:16: error: cannot find symbol
[ERROR]     SampleDataClass getSample();
[ERROR]     ^
[ERROR]   symbol:   class SampleDataClass
[ERROR]   location: interface MtAmazonDynamoDb
```

Will try out dokka today